src/components/Rocketslist.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Rocketslist = ({
+  rockets, loading, error, handleReserveRocket, handleCancelRocket,
+}) => {
+  if (loading) {
+    return (
+      <div>Loading...</div>
+    );
+  } if (error) {
+    return (
+      <div>
+        Error:
+        {' '}
+        {error}
+      </div>
+    );
+  }
+  return (
+    <div className="rocketsdiv">
+      <div className="separator" />
+      <hr />
+      <ul className="table">
+        {rockets.map((rocket) => (
+          <li key={rockets.id} className="rocketname">
+            <div className="rocketimg">
+              <img src={rocket.flickr_images[0]} alt="rocket" height="125px" width="250px" />
+            </div>
+            <div className="rocketstatus">
+              <span className="rocket">{rocket.name}</span>
+              <span className="descrocket">
+                {rocket.reserved ? (
+                  <span className="status">
+                    Reserved
+                  </span>
+                )
+                  : (
+                    <span className="status1">
+                      Reserved
+                    </span>
```

Comment:
- [ ] Kindly use a different color for the reserved action just as done in the design. This so, for proper user experience.
- Using the same color for explame as you did, does not suite best pacticed for user experince 👍 

### Design:
![image](https://github.com/jcmunav63/space-missions-and-rockets/assets/50990925/8cac82f4-94f5-4d11-93b3-a9be53ce4a94)

### Your work:
![image](https://github.com/jcmunav63/space-missions-and-rockets/assets/50990925/a9d921e9-de32-40e6-beb2-8ec3ff2fc942)